script.js

```diff
@@ -1,4 +1,117 @@
+var state = "INPUT PLAYER 1 NAME"
+var numOfPlayer1Wins = 0
+var numOfPlayer2Wins = 0
+var Player1UserName = ""
+var Player2UserName = ""
+var Player1Arr = []
+var Player2Arr = []
+
+
+
+var diceRoll = function () {
+  var randomInteger = Math.ceil(Math.random() * 6);
+  return randomInteger
+}
+
+var beat_that = function (input) {
```

Comment:
Nice that you refactored the whole game into one function! Do take note of naming conventions - function names should be in camelCase (`beatThat`)